### ISSUE TITLE
If you use a CDN you need an internet connection

### ISSUE THREAD

**Issue by ff6347** [technologiestiftung/exhibition-remote-control#2]
What if there is no internet and we still need to turn on the exhibition? I know bundling all that makes more work but would make the application free of dependencies outside of the network

https://github.com/technologiestiftung/exhibition-remote-control/blob/c0ee643af4ede9e8695de5715ca4a124c8399ca9/public/index.html#L5

**Comment by fabianmoronzirfas** [comment]
Yes. Having a local version is enough. We already hat one outage.
